extensions/amp-ad/0.1/amp-ad-custom.js

```diff
@@ -96,17 +96,19 @@ export class AmpAdCustom extends AMP.BaseElement {
             null;
       }
       // Set UI state
-      this.uiHandler.setDisplayState(
-        templateData !== null && typeof templateData == 'object' ?
-        AdDisplayState.LOADED_RENDER_START : AdDisplayState.LOADED_NO_CONTENT);
-      templatesFor(this.win).findAndRenderTemplate(element, templateData)
-        .then(renderedElement => {
-        // Get here when the template has been rendered
-        // Clear out the template and replace it by the rendered version
-          removeChildren(element);
-          element.appendChild(renderedElement);
-          return this;
-        });
+      if (templateData !== null && typeof templateData == 'object') {
+        this.uiHandler.setDisplayState(AdDisplayState.LOADED_RENDER_START);
+        templatesFor(this.win).findAndRenderTemplate(element, templateData)
+          .then(renderedElement => {
+          // Get here when the template has been rendered
+          // Clear out the template and replace it by the rendered version
+            removeChildren(element);
+            element.appendChild(renderedElement);
+            return this;
```

Comment:
I guess the "allow maintainers to make changes" isn't as useful as I hoped! I've just done it.